db/schema.rb

```diff
@@ -559,7 +559,7 @@
     t.integer "user_id"
     t.datetime "returned_at", precision: nil
     t.boolean "billable"
-    t.index "((requested_at)::date), issuer", name: "index_sp_return_logs_on_requested_at_date_issuer", where: "(returned_at IS NOT NULL)"
+    t.index "((returned_at)::date), issuer", name: "index_sp_return_logs_on_returned_at_date_issuer", where: "(billable = true)"
```

Comment:
the "proper" thing to do would be to wait to drop the old index until a second PR, but I figured that this table is so big that it would be valuable to drop the unused index ASAP (keep writes faster) and that the reporting queries should only run at the end of the day, which is usually way after we deploy